addon/styles/partials/_uni-info-box.scss

```diff
@@ -1,12 +1,26 @@
 .uni-info-box {
   @include info-box();
 
+  &__wrapper svg {
+    width: 37px;
+    height: 37px;
+    margin-bottom: 30px;
+  }
+
   &__title {
     @include flex();
     @include align-items-center();
+    font-size: em(22px);
+    color: color("text", "header");
   }
 
   &-mobile-button svg {
     width: 20px;
   }
+
+  &__content {
+    font-size: em(16px);
+    line-height: 1.39;
+    color: color("status-tag", "info-text");
```

Comment:
Hmm... I was not saying this xD This is not a status-tag, therefore shouldn't have a color associated to status tags. I was saying creating this colors on the uniQ, but if this is the only one, I prefer having it hardcoded instead of using someone else's color

Reply:
TBH it was a bit of a knee-jerk reaction... since we'll soon be getting a ticket to redo SS styles including titles, colors, font-weights. But yes, you are absolutely right, speaking out against my own doings, it's better to have the hardcoded color than a wrong key.

Reply:
fix done